internal/handlers/team_members.go

```diff
@@ -0,0 +1,100 @@
+package handlers
+
+import (
+	"backend/internal/middleware"
+	"backend/internal/models"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+)
+
+type TeamMemberHandler struct {
+	db *gorm.DB
+}
+
+func NewTeamMemberHandler(db *gorm.DB) *TeamMemberHandler {
+	return &TeamMemberHandler{db: db}
+}
+
+func (h *TeamMemberHandler) Register(r *gin.RouterGroup) {
+	teamMembers := r.Group("/team-members")
+	{
+		// Public endpoints
+		teamMembers.GET("", h.List)
+		teamMembers.GET("/:id", h.Get)
+
+		// Admin-only endpoints
+		teamMembers.Use(middleware.AdminRequired())
```

Comment:
Same as in your other PR #46, does this get applied only to the routes below it?